src/style.css

```diff
@@ -0,0 +1,54 @@
+* {
+  margin: 0;
+}
+
+.stars {
+  height: 16px;
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  margin-right: 4px;
+  background-position: center;
+}
+
+.stars__star:last-child {
+  margin-right: 0;
+}
+
+.stars--0 .stars__star {
+  background-image: url(images/star.svg);
+}
+
+.stars--1 .stars__star:nth-child(1),
+.stars--2 .stars__star:nth-child(1),
+.stars--3 .stars__star:nth-child(1),
+.stars--4 .stars__star:nth-child(1),
+.stars--5 .stars__star:nth-child(1) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--2 .stars__star:nth-child(2),
+.stars--3 .stars__star:nth-child(2),
+.stars--4 .stars__star:nth-child(2),
+.stars--5 .stars__star:nth-child(2) {
+  background-image: url(images/star-active.svg);
+}
```

Comment:
try to figure out how to accomplish this goal using
:nth-child(-n+2)

Reply:
should be a lot shorter, one line for one class like (.stars--2 .stars__star:nth-child(...your code) {...})